components/developer/golang-116/golang.p5m

```diff
@@ -22,4 +22,4 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
```

Comment:
I noticed that the manifest contains lots of static libraries. Are these really needed? We usually only deliver them if needed.

Reply:
Yes this is golang specific. If we do not ship them the compile fails for anybody except root.